prow/config.yaml

```diff
@@ -1680,6 +1680,30 @@ presubmits:
             limits:
               cpu: 3500m
               memory: 8Gi
+  - name: pull-tekton-results-go-coverage
+    labels:
+      preset-github-token: "true"
+    agent: kubernetes
+    always_run: true
+    rerun_command: "/test pull-tekton-results-go-coverage"
+    trigger: "(?m)^/test (all|pull-tekton-results-go-coverage),?(\\s+|$)"
+    optional: true
+    decorate: true
+    clone_uri: "https://github.com/tektoncd/results.git"
+    spec:
+      containers:
+      - image: gcr.io/tekton-releases/dogfooding/coverage:latest
```

Comment:
We should probably stop using `latest` in future, but all repos do for coverage, so this is fine for now.

Reply:
Created a issue here: #1593